packages/auto-label/test/auto-label.ts

```diff
@@ -339,6 +419,56 @@ describe('auto-label', () => {
       });
       ghRequests.done();
     });
+
+    it('will add a samples tag for a samples repo', async () => {
+      const ghRequests = nock('https://api.github.com')
```

Comment:
Gosh these nock chains are painful. I don't have a better answer, but we should put our heads together with @bcoe to maybe consider a different testing strategy (after this PR lands of course)